NEWS.md

```diff
@@ -1,5 +1,10 @@
 # esqlabsR (development version)
 
+### BUG FIXES
+
+- Fixes a bug where scenario results could not be saved or loaded when scenario name
+contains a slash or a backslash (https://github.com/esqLABS/esqlabsR/issues/547)
```

Comment:
```suggestion
contains a slash or a backslash (#547)
```

Reply:
But for someone who is looking at the NEWS of an installed package, it will not be obvious where to finde the issue?

Reply:
It will be turned into link by pkgdown. like this: https://ggplot2.tidyverse.org/news/index.html#ggplot2-342